plugins/home/src/homePageComponents/Toolkit/Content.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Link } from '@backstage/core-components';
+import {
+  makeStyles,
+  List,
+  ListItemIcon,
+  ListItemText,
+} from '@material-ui/core';
+import React from 'react';
+
+import { useTools } from './Context';
+
+const useStyles = makeStyles(theme => ({
+  toolkit: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  tool: {
+    margin: theme.spacing(0.5, 1),
+  },
+  label: {
+    '& span': {
+      marginTop: theme.spacing(1),
+      fontSize: '0.9em',
+      lineHeight: '1.25',
+      color: theme.palette.text.secondary,
+    },
+  },
+  icon: {
+    width: '64px',
+    height: '64px',
+    borderRadius: '50px',
```

Comment:
Do we have border-radius values defined in the theme? If so, it might be good to use it to maintain consistency.

Reply:
hmm not what I can see, just the color of the border, but we do not want a color here. Could be that I'm missing something, feel free to point me to any example.

Reply:
Here the [explore](https://v4.mui.com/customization/default-theme/#explore), shape property?
![image](https://user-images.githubusercontent.com/6290749/145992652-5f99da5d-d88c-40c1-86e9-5f08bcd0add6.png)

Reply:
Yes, `4px`, is the standard material UI size for components with rounded border, but I'm not sure we want the same size so I'll leave the choice up to you 🙂

Reply:
Probably doesn't make sense for the icon thinking now, disregard what I said haha...

Reply:
oh yes, I think not in this case, as we want it to be a circle! Thanks for the input though!

Reply:
`50%` works always then!